.github/workflows/local.yml

```diff
@@ -42,8 +42,15 @@ jobs:
       uses: mxschmitt/action-tmate@v2
 
     - name: Install Dependencies of GraphScope
+      env:
+        OS : ${{ matrix.os }}
       run: |
-        /bin/bash ${GITHUB_WORKSPACE}/scripts/deploy_local.sh install_deps --verbose
+        if [ "${OS}" == "macOS-latest" ];
+        then
+          /bin/bash ${GITHUB_WORKSPACE}/scripts/deploy_local.sh install_deps --verbose --ci
```

Comment:
???
in case you need, check the env in your script.
https://docs.github.com/en/actions/reference/environment-variables#default-environment-variables